internal/tasks/manifests.go

```diff
@@ -615,6 +621,8 @@ func (j *Janitor) doVulnerabilityCheck(account keppel.Account, repo keppel.Repos
 		return nil
 	}
 
+	vulnInfo.CheckStartedAt = j.timeNow()
```

Comment:
This can also be set right before calling `doVulnerabilityCheck()`, and then it can be a method-local variable instead of appearing in the DB model type.